auth0_flutter_platform_interface/lib/src/auth/auth_passwordless_login_options.dart

```diff
@@ -0,0 +1,32 @@
+import '../request/request_options.dart';
+import 'auth_passwordless_type.dart';
+
+class AuthPasswordlessLoginOptions implements RequestOptions {
```

Comment:
Are we using the same options type for all the passwordless method calls? There's two distinct calls:
1. The call to start passwordless -> this one needs `passwordlessType`
2. The call to login with the received code -> this one does not need `passwordlessType`

Maybe we can have two separate types, one for each call?